presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaTopicFieldGroup.java

```diff
@@ -30,16 +30,19 @@
 {
     private final String dataFormat;
     private final Optional<String> dataSchema;
+    private final Optional<String> subject;
     private final List<KafkaTopicFieldDescription> fields;
 
     @JsonCreator
     public KafkaTopicFieldGroup(
             @JsonProperty("dataFormat") String dataFormat,
             @JsonProperty("dataSchema") Optional<String> dataSchema,
+            @JsonProperty("decoderParams") Optional<String> subject,
             @JsonProperty("fields") List<KafkaTopicFieldDescription> fields)
     {
         this.dataFormat = requireNonNull(dataFormat, "dataFormat is null");
         this.dataSchema = requireNonNull(dataSchema, "dataSchema is null");
+        this.subject = requireNonNull(subject, "subject is null");
```

Comment:
Can we could them as a `Map<String, Object>` ? So in future other related properties can be passed without modifying these classes